README.md

```diff
@@ -175,21 +172,17 @@ database does not have the private key:
 
 ## API
 
-### ssb = require('ssb-db/create')(path)
+### SecretStack.use(require('ssb-db')) => sbot
 
-Create an ssb-db database at the given path, returns an instance.
+Adds ssb-db persistence to a secret-stack setup.
+Without other plugins, this instance will not have replication
+or querying. Loading ssb-db directly is useful for testing,
+but it's recommended to instead start from a plugin bundle like [ssb-server](https://github.com/ssbc/ssb-server)
 
-### require('ssb-db')(db, opts)
+Because of legacy reasons, all the ssb-db methods are mounted on the top level object,
+so it's `sbot.get` instead of `sbot.db.get` as it would be with all the other `ssb-*` plugins.
```

Comment:
Do methods also bind to `sbot.db`, or only to the top level object? Should I open an issue to have it bind to both, in the eventual hope we can migrate to the non-legacy binding?

Reply:
they bind to the top level. yeah I was thinking we should move them to sbot.db and then have a thing that makes it backwards compatible (but print a deprecation message)